openapi/components/schemas/AML/Settings/AmlSettingsPriority.yaml

```
@@ -1,5 +1,7 @@
 type: object
-description: Priority settings.
+description: |-
+  Priority level settings for AML source matches. \
```

Is `\` really needed here?

```suggestion
  Priority level settings for AML source matches.
```

---

That's meant to be there, yes.

---

It adds a new line